### PR TITLE
Return lease status to exporter

### PR DESCRIPTION
**Pull request by NickCao**

### CODE REVIEW

File: internal/service/controller_service.go [hunk 1]
```diff
@@ -302,14 +304,67 @@ func (s *ControllerService) Listen(req *pb.ListenRequest, stream pb.ControllerSe
...
+	if err != nil {
+		logger.Error(err, "failed to watch exporter")
+		return err
+	}
+
+	defer watcher.Stop()
+	for result := range watcher.ResultChan() {
+		switch result.Type {
+		case watch.Added, watch.Modified, watch.Deleted:
+			exporter = result.Object.(*jumpstarterdevv1alpha1.Exporter)
```
Comment (mangelajo) [inline internal/service/controller_service.go:337]: Right, when you subscribe to a watcher, you get an initial update with the current state. Perfect.

**Review by mangelajo — APPROVED** [review]
We just need to update the branch back :)

File: buf.gen.yaml [hunk 2]
```diff
@@ -13,4 +13,5 @@ plugins:
     opt: paths=source_relative
 inputs:
   - git_repo: https://github.com/jumpstarter-dev/jumpstarter-protocol.git
+    branch: lease-status-to-exporter
```
Comment (mangelajo) [inline buf.gen.yaml:16]:
```suggestion
```

**Comment by mangelajo** [comment]
Did some click-ops to remove the branch, and merging it :)

**Comment by NickCao** [comment]
Regenerated and rebased.

**Pull request merged**